uber/models.py

```diff
@@ -1134,7 +1132,7 @@ def discount(self):
     def age_group_conf(self):
         if self.birthdate:
             day = c.EPOCH.date() if date.today() <= c.EPOCH.date() else sa.localized_now().date()
-            attendee_age = (day - self.birthdate).days / 365.2425
+            attendee_age = (day - self.birthdate).days // 365.2425
```

Comment:
This is actually a fairly serious bug which we never noticed until now.  The way we configure our age groups is to have non-overlapping age ranges.  For example, we have one group which is ``13 - 17`` and another group which is ``18 - 20``.

Because we've been doing floating point division division instead of integer division, people whose ages ended up like ``17.71252`` wouldn't fall under any age category.  There are currently 536 people in our reg database who have a birthdate yet are still listed as "age unknown" as a result of this bug, which I'll fix manually later this evening.